install_config/install/prerequisites.adoc

```diff
@@ -797,6 +797,45 @@ See
 link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Logical_Volume_Manager_Administration/index.html[Logical
 Volume Manager Administration] for more detailed information on LVM management.
 
+[[managing-docker-container-logs]]
+*Managing Docker Container Logs*
+
+If your container's log (*_/var/lib/docker/containers/HASH/HASH-json.log_*) is
+increasing, you can use `json-file` logging driver options to manage this.
+
+For example, `--log-opt max-size` rolls over the size, and you can specify the
+`--log-opt max-file`, which is the maximum number of files that a log is rolled
+over before being discarded. You can use these options with Docker 1.8.2 or
```

Comment:
This could also be written more clearly, I think. The gist I'm getting from this is you'll end up with multiple log files based on the max-size, and then this max-file number controls the maximum number of files that are allowed. If this number is reached, then the oldest file gets deleted as each new file is created, is that correct?